gateway/service/src/main/java/org/eclipse/ditto/gateway/service/endpoints/routes/thingsearch/ThingSearchRoute.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2017 Contributors to the Eclipse Foundation
```

Comment:
This should stay at 2017 - the file creation year.
I referenced the other file, the unit test which was added.

Reply:
oops 😅